airflow/contrib/hooks/ssh_hook.py

```diff
@@ -146,13 +162,24 @@ def tunnel(self, local_port, remote_port=None, remote_host="localhost"):
         :type remote_host: str
         :return:
         """
-        tunnel_host = "{0}:{1}:{2}".format(local_port, remote_host, remote_port)
-        proc = self.Popen(["-L", tunnel_host, "echo -n ready && cat"],
-                          stdin=subprocess.PIPE, stdout=subprocess.PIPE)
 
+        import subprocess
```

Comment:
it can, but i find that using subprocess for tunnel makes more sense and works seamlessly with earlier ssh tunnel functionality where you only have to create tunnel once.